package.json

```diff
@@ -15,5 +17,31 @@
   "bugs": {
     "url": "https://github.com/mkhanibeik/nodejs-library/issues"
   },
-  "homepage": "https://github.com/mkhanibeik/nodejs-library#readme"
+  "homepage": "https://github.com/mkhanibeik/nodejs-library#readme",
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "mongoose": "^5.9.12",
+    "nodemon": "^2.0.3"
```

Comment:
Ich glaube, nodemon kann in die devDependencies verschoben werden. Für production braucht man nodemon nicht, denke ich.

Reply:
ja ich denke du hast Recht 👍